src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.security.SecureRandom;
+
 public class ColorSupplier {
+    private static final int colorLength = Colors.values().length;
+    private final int randomIndex = new SecureRandom().nextInt(colorLength);
```

Comment:
`colorLength` is not used, IDEA should have highlight that for you
with your second field you will be having the same color each time, `SecureRandom` itself should be a var not a number it generates